js/webpack.config.js

```diff
@@ -41,18 +41,18 @@ module.exports = {
         new CopyPlugin([
             {
                 from: '*.css',
-                context: 'node_modules/katex/dist/',
+                context: 'node_modules/blatex/dist/',
                 to: '../../wwwroot/css/katex'
             },
             {
                 from: 'fonts',
-                context: 'node_modules/katex/dist/',
+                context: 'node_modules/blatex/dist/',
                 to: '../../wwwroot/css/katex/fonts'
             },
         ],
         )],
-    name: 'blatex',
+    name: 'blatex_wrapper',
     entry: './index.js',
-    output: { filename: outputDir + '/blatex.js' },
+    output: { filename: outputDir + '/blatex_wrapper.js' },
```

Comment:
I'l called `blatex_wrapper` because I was getting the empty object out of `require(..)`, which according to the internet is a circular reference. I sought for an hours and to make this name unambiguous with the blatex package is renamed it.